docs/zh/latest/plugins/mqtt-proxy.md

```diff
@@ -45,7 +45,8 @@ title: mqtt-proxy
 | -------------- | ------- | ------ | ------ | ------ | ------------------------------------------------------ |
 | protocol_name  | string  | 必须   |        |        | 协议名称，正常情况下应为“ MQTT”                        |
 | protocol_level | integer | 必须   |        |        | 协议级别，MQTT `3.1.*` 应为 `4` ，MQTT `5.0` 应是`5`。 |
-| upstream.ip    | string  | 必须   |        |        | 将当前请求转发到的上游的 IP 地址                       |
+| upstream.host  | string  | 必须   |        |        | 将当前请求转发到的上游的 IP 地址或域名                  |
+| upstream.ip    | string  | 必须   |        |        | 推荐使用“host”代替。将当前请求转发到的上游的 IP 地址                       |
```

Comment:
since we only need one of them, naming both of them 'required' might lead to misunderstanding